javers-core/src/test/groovy/org/javers/core/JaversRepositoryE2ETest.groovy

```diff
@@ -1345,4 +1345,53 @@ class JaversRepositoryE2ETest extends Specification {
             assert javers.findSnapshots(QueryBuilder.byInstanceId(it, SnapshotEntity).build()).size() == 1
         }
     }
-}
\ No newline at end of file
+
+    def "should query for commit property starting with partial text"() {
```

Comment:
these 3 tests are very similar, all 3 cases could be tested in a single data test with `where:` and `@Unroll`